File: ContentUpdateAI.EF6/Content/ContentModel.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ContentUpdateAI.EF6.Content
{
    public class ContentModel
    {
        [Key]
        public int ContentId { get; set; }
        public int LineNumber { get; set; }
        public string Content { get; set; }
        [NotMapped]
        public string ChangeToContent { get; set; }
    }
}

File: ContentUpdateAI.App/app/src/app/models/contentModel.ts
export class Content {
  LineNumber: number;
  ContentId: number;
  Content: string;
  ChangeToContent: string;
  constructor(lineNumber: number, content: string, contentId: number, changeToContent: string = "") {
    this.Content = content;
    this.ContentId = contentId;
    this.LineNumber = lineNumber;
    this.ChangeToContent = changeToContent;
  }
}
